generators/Program.cs

```diff
@@ -20,14 +21,29 @@ private static void SetupLogger()
                 .CreateLogger();
         }
 
-        private static void Generate(Options options)
+        private static void RegenerateTestClasses(Options options)
         {
-            Log.Information("Generating tests...");
+            Log.Information("Re-generating test classes...");
+
+            var canonicalDataParser = CreateCanonicalDataParser(options);
```

Comment:
Good point. I think I'll have to catch exceptions in the Program class, to catch and log *any* exception.